typescript/sdk/src/middleware/deploy.ts

```diff
@@ -112,9 +121,68 @@ export class InterchainAccountDeployer extends MiddlewareRouterDeployer<
   constructor(
     multiProvider: MultiProvider,
     configMap: ChainMap<InterchainAccountConfig>,
-    create2salt = 'accountsrouter',
+    create2Salt = 'accountsrouter',
   ) {
-    super(multiProvider, configMap, interchainAccountFactories, create2salt);
+    super(multiProvider, configMap, interchainAccountFactories, create2Salt);
+  }
+
+  async deployContracts(
+    chain: ChainName,
+    config: InterchainAccountConfig,
+  ): Promise<InterchainAccountContracts> {
+    // adapted from super.deployProxy to use a dummy implementation
```

Comment:
This is all weird enough that it deserves a longer comment explaining what were doing and especially *why* we're doing it